### ISSUE TITLE
bbox_loss

### ISSUE THREAD

**Issue by wangjian16** [msracver/Deformable-ConvNets#152]
bbox_loss_ = bbox_weight * mx.sym.smooth_l1(name='bbox_loss_', scalar=1.0, data=(bbox_pred - bbox_target))
bbox_loss = mx.sym.MakeLoss(name='bbox_loss', data=bbox_loss_, grad_scale=1.0 / cfg.TRAIN.BATCH_ROIS)

I used a training set, it is uneven distribution of the number of objects, some of the images have only one object, but, there are dozens of object in others . I want to ask you how to set  the regression's parameter  ' grad_scale ' ,So ,it should be associated with the number of positive samples, rather than a set of constants in advance

**Comment by wangjian16** [comment]
This seems to be a Mxnet symbol programming problem.

thank youvery much

**Comment by larsoncs** [comment]
"grad_scale" param is used to  adjust  the multi-loss  of   wieght,  it  could be seted by constants,  i  think so.
